scheduler.rst

```diff
@@ -211,6 +215,34 @@ For example::
     Check out the `crontab.guru website`_ if you need help to construct/understand
     cron expressions.
 
+Another way of declaring cron triggers is to use the
+:class:`Symfony\\Component\\Scheduler\\Attribute\\AsCronTask` attribute
+on an invokable class::
+
+    // src/Scheduler/Task/SendDailySalesReports.php
+    namespace App\Scheduler\Task;
+
+    use Symfony\Component\Scheduler\Attribute\AsCronTask;
+
+    #[AsCronTask('0 0 * * *')]
+    class SendDailySalesReports
+    {
+        public function __invoke()
+        {
+            // ...
+        }
+    }
+
+If you want to use another method name than ``__invoke()``, you can specify it
```

Comment:
I think you should also document `timezone`, `jitter`, `arguments`, `schedule`

Reply:
Added some examples 🙂